docs/events/README.md

```diff
@@ -1,5 +1,9 @@
 # SkyWalking Events
 
+## Welcome Haochao Zhuang as new committer
```

Comment:
Remove the oldest news to keep it only 3 in the home page, same as Chinese version